src/latte/functional/disentanglement/mutual_info.py

```diff
@@ -277,6 +282,12 @@ def dlig(
     """
     Calculate Dependency-Aware Latent Information Gap (DLIG) between latent vectors and attributes
 
+    Dependency-aware Latent Information Gap (DLIG) is a latent-centric counterpart to DMIG. DLIG evaluates disentanglement of a set of semantic attributes :math:'{a_i}' with respect to a latent dimension :math:'z_d' such that
+
+    .. math:: \operatorname{DLIG}({a_i}, z_d) = \dfrac{\mathcal{I}(a_j, z_d)-\mathcal{I}(a_k, z_d)}{\mathcal{H}(a_j|a_k)},
+
+    where :math:`j=\operatorname{arg}\max_i \mathcal{I}(a_i, z_d)`, :math:`k=\operatorname{arg}\max_{i≠j} \mathcal{I}(a_i, z_d)`.
```

Comment:
Mutual info and entropy need to be defined again for each function. Each function doc should be as standalone as possible.